api/lua/pinnacle/signal.lua

```diff
@@ -97,6 +99,30 @@ local signals = {
         ---@type fun(response: table)
         on_response = nil,
     },
+    WindowLayoutModeChanged = {
+        ---@type grpc_client.h2.Stream?
+        sender = nil,
+        ---@type { callback_id: integer, callback: fun(window: pinnacle.window.WindowHandle, layout_mode: pinnacle.window.v1.LayoutMode) }[]
```

Comment:
The type for `layout_mode` should be `pinnacle.util.v1.LayoutMode`

Reply:
Thanks, fixed